src/client/inputhandler.h

```diff
@@ -129,6 +131,15 @@ class MyEventReceiver : public IEventReceiver
 		return b;
 	}
 
+	// Checks whether a key was released and resets the state
+	bool WasKeyReleased(const KeyPress &keyCode)
```

Comment:
This is analog to https://github.com/minetest/minetest/pull/5699/files#diff-b5b39c3a29359759ac6f0293dc37a453R125.

I think changing the function name here would be inconsistent.